src/BuildScriptGenerator/Ruby/RubyBashBuildSnippet.sh.tpl

```diff
@@ -19,29 +19,34 @@ echo
     gem install bundler
 {{ end }}
 
-{{ if CustomBuildCommand | IsNotBlank }}
-	echo
-	echo "Running '{{ CustomBuildCommand }}'..."
-	echo
-	{{ CustomBuildCommand }}
-{{ else }}
-    {{ if ConfigYmlFileExists }}
-        {{ if GemfileExists }}
-        echo "Running 'bundle install'..."
-        echo
-        bundle install
+if [ ! -z "$RUN_BUILD_COMMAND" ]
+then
+	echo 'Running $RUN_BUILD_COMMAND'
+	${RUN_BUILD_COMMAND}
+else
+    {{ if CustomBuildCommand | IsNotBlank }}
```

Comment:
Do you have an example of a `CustomBuildCommand` that you can provide? Would it include the `RUN_BUILD_COMMAND` and that's why we don't want them in the same flow? Just making sure I understand the purposes of the different variables.